### PR TITLE
training update changes

### PR DESCRIPTION
**Pull request by prishavall**
# Title of Pull Request HERE

**What user problem are we solving?**
Migrating unit tests from backend to the training module.

**What solution does this PR provide?**
Fixes the pathway errors and import changes with the transfer from backend to training.
**Testing Methodology**
How did you test your changes and verify that existing 
functionality is not broken
Ran tests individually
**Any other considerations**
One test model_py is still not working need to look into django implementation for that.

### CODE REVIEW

**Comment by dwu359** [comment]
@prishavall Update PR title in both the post and the PR itself so it's easier to find your PR in github projects

**Comment by dwu359** [comment]
@prishavall Also include the specific commands needed to test your PR

File: training/training/core/dl_model.py [hunk 1]
```diff
@@ -43,6 +43,7 @@ def fromLayerParamsList(cls, layer_params_list: list[LayerParams]):
     def build_model(self, layer_list):
         model = nn.Sequential()
         ctr = 1
+        print("Hello World")
```
Comment (farisdurrani) [inline training/training/core/dl_model.py]: nit: remove print statements

File: training/tests/test_loss_function.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import pytest
```
[the remark this answers is not in the document]
Comment (farisdurrani) [reply training/tests/test_loss_function.py:1]: Ignore these pyright import errors

**Pull request closed without merging**